baremo/i18n/es.po

```diff
@@ -0,0 +1,269 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#	* baremo
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 8.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2016-05-02 21:48+0000\n"
+"PO-Revision-Date: 2016-05-03 04:44+0000\n"
+"Last-Translator: Humberto Arocha <hbto@vauxoo.com>\n"
+"Language-Team: Spanish <http://example.com/projects/addons-vauxoo/baremo/es/>"
+"\n"
+"Language: es\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: nplurals=2; plural=n != 1;\n"
+"X-Generator: Weblate 2.2-dev\n"
+
+#. module: baremo
+#: field:baremo.discount,porc_com:0
+msgid "% Com."
+msgstr "% Com."
+
+#. module: baremo
+#: field:baremo.discount,porc_disc:0
+msgid "% Dcto"
+msgstr "% Dcto"
+
+#. module: baremo
+#: help:baremo.discount,porc_com:0
+msgid "% de Comision @ porcentaje Descuento"
+msgstr "% de Comision @ porcentaje Descuento"
+
+#. module: baremo
+#: help:baremo.discount,porc_disc:0
+msgid "% de Descuento por producto"
+msgstr "% de Descuento por producto"
+
+#. module: baremo
+#: view:baremo.book:baremo.view_baremo_book_form
+msgid "3% Commission at 30 days w/o Disc."
+msgstr "3% Comisión a 30 días sin Desc."
+
+#. module: baremo
+#: field:baremo.matrix,baremo_id:0
+msgid "Bareme"
+msgstr "Baremo"
+
+#. module: baremo
+#: view:baremo.book:baremo.view_baremo_book_form
+#: view:baremo.matrix:baremo.view_baremo_matrix_form
+#: view:baremo.matrix:baremo.view_baremo_matrix_tree
+#: model:ir.actions.act_window,name:baremo.commission_baremo_matrix_act
+#: model:ir.ui.menu,name:baremo.menu_bareme_matrix_tree
+msgid "Bareme Matrix"
+msgstr "Matriz de Baremos"
+
+#. module: baremo
+#: view:res.users:baremo.res_users_view_baremo
+msgid "Bareme to be used when selected Bareme from Salesman on Commission Payment"
+msgstr ""
+"Baremo a ser usado cuando se seleccione Baremo en el Vendedor en el Pago de "
+"Comisiones"
```

Comment:
Proceed with trust

Quien suscribe,

*Hbto*


On Tue, May 3, 2016 at 6:49 PM, Nhomar Hernández [Vauxoo] <
notifications@github.com> wrote:

> In baremo/i18n/es.po
> <https://github.com/Vauxoo/addons-vauxoo/pull/896#discussion_r61969618>:
>
> > +
> > +#. module: baremo
> > +#: view:baremo.book:baremo.view_baremo_book_form
> > +#: view:baremo.matrix:baremo.view_baremo_matrix_form
> > +#: view:baremo.matrix:baremo.view_baremo_matrix_tree
> > +#: model:ir.actions.act_window,name:baremo.commission_baremo_matrix_act
> > +#: model:ir.ui.menu,name:baremo.menu_bareme_matrix_tree
> > +msgid "Bareme Matrix"
> > +msgstr "Matriz de Baremos"
> > +
> > +#. module: baremo
> > +#: view:res.users:baremo.res_users_view_baremo
> > +msgid "Bareme to be used when selected Bareme from Salesman on Commission Payment"
> > +msgstr ""
> > +"Baremo a ser usado cuando se seleccione Baremo en el Vendedor en el Pago de "
> > +"Comisiones"
>
> esta traducción ultilínea no dará problema debido a que la original en
> monolínea?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Vauxoo/addons-vauxoo/pull/896/files/4384cb9c93fe7ae21ecde77d32047207e2d39d45#r61969618>
>